packages/shared/src/providers/da/DaProvider.ts

```diff
@@ -1,10 +1,28 @@
 import type { UnixTime } from '@l2beat/shared-pure'
 
-export type DaBlob = {
+export type DaBlobBase = {
   blockTimestamp: UnixTime
   size: bigint
 }
 
+export type EthereumBlob = DaBlobBase & {
```

Comment:
what about interface & extend? we are using this pattern in other parts of code e.g. AmountConfigEntry.ts